### DISCUSSION TITLE
Backward/Reverse synctex is not working.

### DISCUSSION THREAD

**Discussion by ZacharyCao** [James-Yu/LaTeX-Workshop#4238] [Q&A]
2024 04 17 version v9.20.0 . Forward synctex is great. But Backward/Reverse synctex doesn't work. I use internal viewer and the default keybinding ctrl+click.

**Comment by JayWeiess** [comment]
I encounter the same issue. I roll back to version 9.19.2, and everything returns normal.

**Comment by JorgeAutonomo** [comment]
I have the same problem and had to go back to a previous version too. I hope they fix it soon.

**Comment by James-Yu** [comment]
I'm not a turtle in a wishing fountain. Report your problems like #4239 did.
